components/Footer/Footer.tsx

```diff
@@ -0,0 +1,83 @@
+import Link from "next/link"
+import React from "react"
+import styles from "./footer.module.css"
+
+const Footer = () => {
+	return (
+		<div className={styles.footer}>
+			<div className={styles.footer__top}>
+				<ul className={styles.footer__box}>
+					<li className={styles.footer__linkName}>Community</li>
+					<li className={styles.footer__linkItems}>
+						<a
+							target="_blank"
+							rel="noreferrer"
```

Comment:
remove these two please. To be accessible links should not open in a new browser